eslint.config.cjs

```diff
@@ -1,111 +1,125 @@
+        ecmaFeatures: {
+          jsx: true
+        }
       },
+      globals: {
+        React: 'readonly',
+        window: 'readonly',
+        document: 'readonly',
+        global: 'readonly',
+        HTMLElement: 'readonly',
+      }
+    },
+    plugins: { 
+      '@typescript-eslint': ts, 
+      prettier: prettierPlugin,
+      'react-hooks': reactHooks,
+      'react-refresh': reactRefresh,
     },
     rules: {
       'prettier/prettier': 'error',
-      'import/no-unresolved': 'error',
       '@typescript-eslint/quotes': [
         'error',
         'single',
-        { avoidEscape: true, allowTemplateLiterals: false },
-      ],
-      'import/no-duplicates': 'error',
-      'import/extensions': ['error', 'ignorePackages', { ts: 'never', tsx: 'never' }],
-      'no-console': 'error',
-      'import/order': [
-        'error',
-        {
-          groups: [
-            'builtin',
-            'external',
-            'internal',
-            'parent',
-            'sibling',
-            'index',
-            'object',
-            'type',
-          ],
-          'newlines-between': 'always',
-          alphabetize: { order: 'asc', caseInsensitive: true },
+        { 
+          avoidEscape: true, 
+          allowTemplateLiterals: false 
         },
       ],
-
-      // Preferir reglas de TS sobre las base
+      'no-console': 'warn',
       'no-unused-vars': 'off',
-      // '@typescript-eslint/no-shadow': 'error', // activar si lo necesitás
-      // Ajustes comunes de Airbnb (opcionales)
-      // 'no-plusplus': 'off',
-      // 'no-underscore-dangle': 'off',
-      // 'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['acc', 'e', 'ctx', 'req', 'res', 'state'] }],
+      '@typescript-eslint/no-unused-vars': [
+        'warn',
+        { 
+          argsIgnorePattern: '^_',
+          varsIgnorePattern: '^_',
+        }
+      ],
+      '@typescript-eslint/no-explicit-any': 'warn',
+      '@typescript-eslint/explicit-module-boundary-types': 'off',
+      '@typescript-eslint/no-non-null-assertion': 'warn',
+      'no-undef': 'off', // TypeScript handles this
+      
+      // React Hooks rules
+      'react-hooks/rules-of-hooks': 'error',
+      'react-hooks/exhaustive-deps': 'warn',
+      
+      // React Refresh
+      'react-refresh/only-export-components': [
+        'warn',
+        { allowConstantExport: true },
+      ],
+    },
+  },
+
+  // Storybook files - more lenient rules
+  {
+    files: ['**/*.stories.{ts,tsx}'],
+    rules: {
+      'no-console': 'off',
+      '@typescript-eslint/no-explicit-any': 'off',
+      'react-refresh/only-export-components': 'off',
     },
   },
-  ...astro.configs['flat/recommended'],
+
+  // Test files - more lenient rules
+  {
+    files: ['**/*.test.{ts,tsx}', '**/*.spec.{ts,tsx}', '**/test-utils.{ts,tsx}', '**/setup.{ts,tsx}'],
+    rules: {
+      'no-console': 'off',
+      '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
For test files you also disable `@typescript-eslint/no-explicit-any`. This again contradicts the project's 'never use any' rule.

**Sugerencia:**
```typescript
Avoid disabling `no-explicit-any` for all tests. If necessary for some test utilities, only disable per-file or per-line and provide a justification comment. Prefer `unknown` or typed mocks instead of `any`.
```